group-coordinator/src/main/resources/common/message/ConsumerGroupMemberMetadataValue.json

```diff
@@ -35,6 +35,20 @@
     { "name": "RebalanceTimeoutMs", "type": "int32", "versions": "0+", "default": -1,
       "about": "The rebalance timeout" },
     { "name": "ServerAssignor", "versions": "0+", "nullableVersions": "0+", "type": "string",
-      "about": "The server assignor to use; or null if not used." }
+      "about": "The server assignor to use; or null if not used." },
+    { "name": "ClassicMemberMetadata", "versions": "0+", "nullableVersions": "0+", "type": "ClassicMemberMetadata",
```

Comment:
Let's make it a tagged field in order to not break the backward compatibility of the record. You can do it by adding `"taggedVersions": "0+", "tag": 0`.